windows/Shared/JSValueWriterHelper.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <any>
+#include <map>
+#include <string>
+#include <vector>
+
+#include <winrt/Microsoft.ReactNative.h>
+#include <winrt/base.h>
+
+namespace ReactApp
+{
+    void JSValueWriterWriteValue(winrt::Microsoft::ReactNative::IJSValueWriter const &writer,
```

Comment:
This looks like its basically the same as:

#include <JSValueWriter.h>
winrt::Microsoft::ReactNative::WriteValue(writer, value);